extensions/amp-story/0.1/amp-story-share.js

```diff
@@ -307,20 +313,51 @@ export class ShareWidget {
     this.loadRequiredExtensions();
 
     this.requestService_.loadBookendConfig().then(config => {
-      const providers = config && config['share-providers'];
+      // TODO(#14591): Remove when old version is deprecated.
```

Comment:
It would be useful to clarify what the "old version" refers to.

Reply:
+1, here and elsewhere in the PR